Swifter.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "Swifter"
-  s.version      = "2.4.0"
+  s.version      = "2.4.1"
```

Comment:
Given that we're changing quite a bit and adding new API, please don't change the pod spec. I'll handle a minor version bump separately.

Reply:
I understood and put it back.